code/datums/mutations.dm

```diff
@@ -318,6 +318,56 @@ var/thanks_tobba = 'icons/fonts/runescape_uf.ttf'
 		return
 	owner.disabilities &= ~CLUMSY
 
+/datum/mutation/human/cluwne
+
+	name = "Cluwne"
+	quality = NEGATIVE
+	text_gain_indication = "<span class='danger'>You feel like your brain is tearing itself apart.</span>"
+
+/datum/mutation/human/cluwne/on_acquiring(mob/living/carbon/human/owner)
+	if(..())
+		return
+	owner.disabilities |= CLUMSY
+	owner.adjustBrainLoss(200)
+
+	var/mob/living/carbon/human/H = owner
+
+	if(!istype(H.wear_mask, /obj/item/clothing/mask/gas/clown_hat/cluwne))
+		if(!H.unEquip(H.wear_mask))
+			qdel(H.wear_mask)
+		H.equip_to_slot_or_del(new /obj/item/clothing/mask/gas/clown_hat/cluwne(H), slot_wear_mask)
+	if(!istype(H.wear_mask, /obj/item/clothing/under/rank/clown/cluwne))
+		if(!H.unEquip(H.w_uniform))
+			qdel(H.w_uniform)
+		H.equip_to_slot_or_del(new /obj/item/clothing/under/rank/clown/cluwne(H), slot_w_uniform)
+	if(!istype(H.shoes, /obj/item/clothing/shoes/clown_shoes/cluwne))
+		if(!H.unEquip(H.shoes))
+			qdel(H.shoes)
+		H.equip_to_slot_or_del(new /obj/item/clothing/shoes/clown_shoes/cluwne(H), slot_shoes)
+
+	owner.equip_to_slot_or_del(new /obj/item/clothing/gloves/color/white(owner), slot_gloves) // this is purely for cosmetic purposes incase they aren't wearing anything in that slot
+	owner.equip_to_slot_or_del(new /obj/item/weapon/storage/backpack/clown(owner), slot_back) // ditto
+
+/datum/mutation/human/cluwne/on_life(mob/living/carbon/human/owner)
+	if((prob(15) && owner.paralysis <= 1))
+		owner.adjustBrainLoss(200) // don't want to code special manitol snowflake interactions
```

Comment:
1)doesn't 200 brain DMG Jill
2)shouldn't you set it rather than adjusting otherwise it'll stack